addons/mail/static/src/core/common/store_service.js

```diff
@@ -200,6 +200,15 @@ export class Store extends BaseStore {
 
     cannedReponses = this.makeCachedFetchData({ canned_responses: true });
 
+    specialMentions = [
+        {
+            label: "everyone",
+            channel_types: ["channel", "group"],
```

Comment:
Why limit it like that?

Reply:
It does not really make sense to mention `@everyone` in a chat (you can't on Discord) and the concept of channel members don't exist in the chatter.